src/Raven.NewClient/Http/RequestExecuter.cs

```diff
@@ -205,10 +205,10 @@ public void Execute<TResult>(RavenCommand<TResult> command, JsonOperationContext
                         // we intentionally don't dispose the reader here, we'll be using it
                         // in the command, any associated memory will be released on context reset
                         var blittableJsonReaderObject = await context.ReadForMemoryAsync(stream, "PutResult");
-                        if (response.Headers.ETag != null)
+                        if (response.Headers.Contains("Etag"))
```

Comment:
There is actually a reason for the requirement for quotes.
https://www.w3.org/Protocols/rfc2616/rfc2616-sec3.html#sec3.11
So we have to actually send the quotes